components/event-bus/internal/knative/subscription/controller/subscription/reconcile.go

```diff
@@ -130,9 +151,11 @@ func (r *reconciler) reconcile(ctx context.Context, subscription *eventingv1alph
 		}
 	} else {
 		// The object is being deleted
+		KymaSubscriptionsGauge.DeleteKymaSubscriptionsGauge(subscription.Namespace, subscription.Name)
```

Comment:
Naming is not meaningful. Kyma subscription gauge is deleting itself?

Reply:
OK, will rename